src/web-animations-next-player.js

```diff
@@ -30,6 +30,38 @@
 
   // TODO: add a source getter/setter
   scope.Player.prototype = {
+    _updateChildTiming: function(childPlayer, offset) {
+      if (childPlayer.startTime != this.startTime + offset) {
+        if (this.startTime === null) {
+          childPlayer.currentTime = this.source.player.currentTime - offset;
+          childPlayer._startTime = null;
+        } else {
+          childPlayer.startTime = this.startTime + offset;
+        }
+      }
+      // FIXME: What is this for?
```

Comment:
OK, can you add a test when the group is playing in reverse with 'none' fill, and the child plays past its beginning (i.e. unfills)? If that passes without this bit of code, nuke the code.